src/popup/popup.js

```
@@ -39,47 +40,92 @@ let bmFileType
 
 
 // URL matchers for different sections and rule types
-const URL_MATCHERS = {
+let URL_MATCHERS = {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'URL_MATCHERS' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)